webapp/components/user_settings/custom_theme_chooser.jsx

```diff
@@ -291,7 +314,7 @@ class CustomThemeChooser extends React.Component {
                     </div>
                 );
 
-                colors += theme[element.id] + ',';
+                colors += formatMessage(messages[element.id]) + ': ' + theme[element.id] + ', ';
```

Comment:
Why are we using the localized strings? Will this work when you switch to a non-english language? When happens when you copy and paste from another language?

@esethna can we test again with different languages?